package.json

```diff
@@ -50,24 +50,19 @@
     "webpack-merge": "^4.0.0"
   },
   "engines": {
-    "node": ">= 4.3 < 5.0.0 || >= 5.10"
+    "node": ">= 4.8 < 5.0.0 || >= 5.10"
```

Comment:
Breaking change :disappointed:

Reply:
Why is this a breaking change ? But I can revert it in this PR doesn't matter...

Reply:
@michael-ciniawsky let's do it in next major :+1:

Reply:
done